anomalib/deploy/optimize.py

```diff
@@ -65,14 +65,14 @@ def export_convert(
         input_names=["input"],
         output_names=["output"],
     )
+    export_path = os.path.join(str(export_path), "openvino")
```

Comment:
I think we should move this line under `export_model == "openvino"` and move the `with open ...` lines to the top. Otherwise it will create meta_data.json under openvino directory. Since the idea is to have it common for both, we can just create the metadata file in `export_path`.

Reply:
can't we set `export_path` as 

```
export_path = os.path.join(str(export_path), export_mode)
```